### PR TITLE
Algorithm rand bindings

### PR DESCRIPTION
**Pull request by leclair-7**
## Description

Binds algorithms/rand to pydp. 

Please mark options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
I ran build PyDP.sh, ran python on my terminal:
import pydp as dp
dp.Geometric()
dp.UniformDouble()

Numbers matched what rand.h indicates they should be.

### CODE REVIEW

File: tests/algorithms/test_rand.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import pytest
+import pydp as dp
+
+
+def test_rand_UniformDouble():
+    ud = dp.UniformDouble()
+    assert isinstance(ud, float)
```
Comment (chinmayshah99) [inline tests/algorithms/test_rand.py:7]: https://github.com/google/differential-privacy/blob/master/differential_privacy/algorithms/rand_test.cc#L52
Can we add all of these tests?

**Review by chinmayshah99 — CHANGES_REQUESTED** [review]

File: tests/algorithms/test_rand.py (hunk 1 above)
Comment (leclair-7) [reply tests/algorithms/test_rand.py:7]: Yeah good idea, I'm putting an isinstance assertion for every instance and function we currently have binded. I figure it's a decent sanity check on return policies.

Comment (chinmayshah99) [reply tests/algorithms/test_rand.py:7]: While with these tests, we get to know that at least it is returning something sensible, we want to make sure that it doesn't return a junk value, hence tests for actual values.

**Comment by leclair-7** [comment]
Ok, I've added instance checks to everything currently binded except utils. For some reason to import it, it has to be "import pydp.pydp.util" which I don't think is intended.

**Review by alejandrosame — DISMISSED** [review]
I approve the PR, but I want to raise awareness that it is bad practice to make PRs (and generally group commits) that span files unrelated to the feature that the PR targets.

In this case, the addition of `isinstance` tests for the other classes should be made in a separate PR to make the scope clearer. It's not critical at all right now, but at some point I expect that we will need to reinforce this kind of workflows.

File: tests/algorithms/test_algorithms.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import pytest
+import pydp as dp
+
+
+def test_algorithms_BoundedMean():
```
Comment (chinmayshah99) [inline tests/algorithms/test_algorithms.py]: This function needs to be shifted to `test_bounded_mean.py`

**Review by chinmayshah99 — CHANGES_REQUESTED** [review]

File: tests/algorithms/test_algorithms.py (hunk 2 above)
Comment (chinmayshah99) [reply tests/algorithms/test_algorithms.py]: hey @leclair-7, I meant moving it to the different file called `test_bounded_mean.py`

Comment (leclair-7) [reply tests/algorithms/test_algorithms.py]: Oh gotcha

Comment (leclair-7) [reply tests/algorithms/test_algorithms.py]: All good now

**Review by chinmayshah99 — APPROVED** [review]
LGTM!

**Pull request merged**